src/widgets/_panes.scss

```diff
@@ -21,4 +21,39 @@ paned.horizontal > separator {
     // This creates the invisible resize handle
     margin-right: rem(-7px);
     min-width: rem(8px);
+
+    &.wide {
+        background-image:
+            linear-gradient(
+                to bottom,
+                rgba(black, 0.15) 1px,
+                rgba(white, 0.01) 1px,
+                rgba(white, 0.01) 2px,
+                transparent 2px
+            );
+    }
```

Comment:
This is an entirely different thing and should be in its own branch

Reply:
Will remove.